### PR TITLE
docs: common errors & supported browsers and frameworks & getting-started (web aspect)

### PR DESCRIPTION
**Pull request by CoderSerio**
1. create 2 docs:
- common errors
- supported browsers and frameworks

2. move 2 picture files to a new directory

### CODE REVIEW

**Comment by CoderSerio** [comment]
Fix #373

**Comment by CoderSerio** [comment]
add a new doc, getting-started(web aspect), which refers to lots of example files.
partially finish #370 .

File: example/dubbo-web-example/doc-images/web.png
Comment (jianyi-gronk) [inline example/dubbo-web-example/doc-images/web.png:1]: There is no need to modify this file

File: example/dubbo-web-example/doc-images/arc.png
Comment (jianyi-gronk) [inline example/dubbo-web-example/doc-images/arc.png:1]: There is no need to modify this file

File: example/dubbo-web-example/README.md
Comment (jianyi-gronk) [inline example/dubbo-web-example/README.md:1]: There is no need to modify this file

**Review by jianyi-gronk — APPROVED** [review]
LGTM.

**Pull request merged**